lib/ui/page/home/page/chat/widget/data_attachment.dart

```diff
@@ -235,11 +235,9 @@ class _DataAttachmentState extends State<DataAttachment> {
                         mainAxisSize: MainAxisSize.min,
                         children: [
                           Text(
-                            'label_kb'.l10nfmt({
-                              'amount': e.original.size == null
-                                  ? 'dot'.l10n * 3
-                                  : e.original.size! ~/ 1024
-                            }),
+                            e.original.size == null
+                                ? 'dot'.l10n * 3
+                                : e.original.size!.asBytes,
```

Comment:
Тут, кстати, кажется, что можно было бы использовать `??`:

```suggestion
                            e.original.size?.asBytes ?? ('dot'.l10n * 3)
```